libraries/targetVideoUtils/bidderUtils.js

```diff
@@ -0,0 +1,184 @@
+import {BANNER, VIDEO} from '../../src/mediaTypes.js';
+import {find} from '../../src/polyfill.js';
+import {getRefererInfo} from '../../src/refererDetection.js';
+import {createTrackPixelHtml, deepAccess, getBidRequest, logError} from '../../src/utils.js';
+import {BANNER_ENDPOINT_URL, VIDEO_ENDPOINT_URL, MARGIN, TIME_TO_LIVE} from './constants.js';
```

Comment:
If the goal is to make these functions useful over multiple adapters, you may want to avoid importing constants unique to you

Reply:
I will update PR today with changes you suggested. Thanks.